### PR TITLE
OWKMeans: fix crashes when switching datasets while algo is running 

### PR DESCRIPTION
**Pull request by kernc**

### CODE REVIEW

**Comment by ajdapretnar** [comment]
With this change I'm getting totally whacked initial widget state. The wrong number of clusters and the plot is not resized properly. Tried on iris.

screen shot 2017-02-27 at 15 03 59

**Comment by kernc** [comment]
Thanks. Please try again.

**Comment by ajdapretnar** [comment]
Better.

**Comment by ajdapretnar** [comment]
Should I ignore Travis or will you ping it again?

**Comment by kernc** [comment]
Of course not. Thanks.

File: orangecontrib/educational/widgets/owkmeans.py [hunk 1]
```diff
@@ -33,11 +33,14 @@ def __init__(self, owkmeans):
     def __del__(self):
         self.wait()
 
+    is_running = True
```
Comment (kernc) [inline orangecontrib/educational/widgets/owkmeans.py]: No problem, please find changed.

**Pull request merged**